### ISSUE TITLE
Server-side rendering

### ISSUE THREAD

**Issue by nikgraf** [gaearon/react-transform-hmr#13]
When you compile your server with `babel-node` and render React Components on the server you will get the following error message: 

```
/Users/nikgraf/Projects/time/node_modules/react-transform-hmr/lib/index.js:54
    throw new Error('locals[0] does not appear to be a `module` object with Hot Module ' + 'replacement API enabled. You should disable react-transform-hmr in ' + 'production by using `env` section in Babel configuration. See the ' + 'example in README: https://github.com/gaearon/react-transform-hmr');
    ^

Error: locals[0] does not appear to be a `module` object with Hot Module replacement API enabled. You should disable react-transform-hmr in production by using `env` section in Babel configuration. See the example in README: https://github.com/gaearon/react-transform-hmr
    at proxyReactComponents (/Users/nikgraf/Projects/time/node_modules/react-transform-hmr/lib/index.js:54:11)
    at Object.<anonymous> (/Users/nikgraf/Projects/time/src/containers/Index.js:5:67)
    at Module._compile (module.js:434:26)
    at normalLoader (/Users/nikgraf/Projects/time/node_modules/babel-core/lib/api/register/node.js:199:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/nikgraf/Projects/time/node_modules/babel-core/lib/api/register/node.js:216:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/nikgraf/Projects/time/src/routes.js:8:35)
22 Sep 21:55:38 - [nodemon] app crashed - waiting for file changes before starting...
```

It's happening due this change: https://github.com/gaearon/react-transform-hmr/commit/9afec1bf687b170ca794fe2cca1622042e85aef7

Ideally you want to avoid transforming on the server. Babel uses the existing `.babelrc` by default. There are several solutions I can imagine:
1. Split the server into the webpack and the render part and proxy the rendering endpoint. In case `babel-node` would allow `babelrc` option you could provide an alternative `.babelrc`.
2. Don't return an error and make it noop again :smile: 

Did I miss something? I'm happy to contribute, but I wasn't sure whats your opinion?

P.S: I went for the proxy solution as I want to run the rendering server with nodemon anyway but avoid webpack from restarting continuously. I worked around it by setting `BABEL_ENV="developer-client"`only  for the webpack process and also use "developer-client" in .babelrc.

**Comment by mykhailo-riabokon** [comment]
Do you have in your server.js file import react routes?

**Comment by gaearon** [comment]
This appears to be a duplicate of https://github.com/gaearon/react-transform-hmr/issues/5. Please read the instructions for server rendering there.

**Comment by gaearon** [comment]
(I think configuring transform in Webpack config as suggested there is sensible.)

**Comment by nikgraf** [comment]
Thanks for the hint!
